README.md

```diff
@@ -70,21 +108,77 @@ popular packages in Nixpkgs, like VSCode.
 ### NixOS
 
 ```nix
-{
-  environment.variables.NIXOS_OZONE_WL = "1";
-}
+environment.variables.NIXOS_OZONE_WL = "1";
```

Comment:
Having the attrset {} solves the syntax error that would otherwise be shown.

Reply:
Eh, thought the block looked like a waste of space. If you want, but everybody has red equal signs in their nix docs.

Reply:
Sure, I don't mind. I just hated how the equals looked. Should be fixed in GitHub's highlighting imo.

Reply:
I'm going to leave this one simply because some of the other snippets have the same problem. If I fix the other snippets too, I would introduce another level of indentation, and it would be missing the context of the rest of the attrs. For example, there is a block that shows a flake's `outputs` but `inputs` is omitted because it is irrelevant to the example, besides being shown previously in the first two examples. Otherwise, I totally get where you're coming from, but I would prioritize consistency over working around the grammar rules that GitHub uses.

Reply:
Sounds alright to me. Is this ready to merge then?